README.md

```diff
@@ -7,9 +7,88 @@ Throughout the project, one of our focuses will be on providing a fluid and resp
 ## Learning Goals
 
 * Gain an understanding of how to write clean HTML and CSS to match a provided comp
+
 * Understand how to implement client-side data persistence using `localStorage`
+
 * Understand what it looks like to have a separate data model (using a class) and DOM model
+
 * Incorporate & iterate over arrays in order to filter what is being displayed
+
 * Craft code with clean style, using small functions that show trends toward DRYness and SRP
 
-The project spec & rubric can be found [here](https://frontend.turing.io/projects/module-1/ideabox-group.html)
+-------------------------------------------------------------------------------
+
+## Overview
+
+![make own idea cards](https://frontend.turing.io/projects/module-1/assets/ideabox-group/desktop.jpg)
+
+-------------------------------------------------------------------------------
+
+### How?  Main Paige Functionality
+
+
+#### Creating  and Deleting Idea Cards
+
+* When both `Title` and `Body` input fields are filled with text `Saved button` is activated.
+
+* Save and Create Idea Card by clicking a `Save` button, idea card displayed on a main page and input fields are cleared.
+
+* When `star` clicked on idea card, card is favourited.
+
+* When `X` clicked idea card, card is deleted.
+
+#### Starring Idea cards
+
+* When `Show Starred Ideas` button is clicked, only idea cards which been favourited will display on a screen. Then button toggles to `Show All Ideas` button.
+
+* When `Show All ideas` button is clicked  all ideas will display on a screen.
+
+#### Searching an Idea Card
+
+* To search through the ideas made for a specific word or phrase, use the `Search Bar`. When typing the phrase, the idea cards will be filtered accordingly.
```

Comment:
This is worded a little funny, but I really can't think of a better way to describe the search bar functionality